action.yml

```diff
@@ -281,19 +329,23 @@ runs:
          fi
 
          cd kernel/${{ inputs.kernel-dir }}
+         if ! grep -qsi "LD *= *ld.lld" Makefile; then
+             if [ -f "$HOME/clang/bin/ld" ]; then
+                 rm -f "$HOME/clang/bin/ld"
+             fi
+         fi   
          function version_gt() { test "$(echo -e "$1\n$2" | sort -V | tail -n1)" == "$1"; }
          VERSION=$(grep -E '^VERSION = ' Makefile | awk '{print $3}')
          PATCHLEVEL=$(grep -E '^PATCHLEVEL = ' Makefile | awk '{print $3}')
-         SUBLEVEL=$(grep -E '^SUBLEVEL = ' Makefile | awk '{print $3}')
```

Comment:
Do not delete it

Reply:
> Do not delete it

？？？

Reply:
https://github.com/dabao1955/kernel_build_action/blob/main/action.yml#L302-L302